script.js

```diff
@@ -292,3 +320,22 @@ function showMangaDetails(manga, categories, source) {
 
     showModal('manga-modal');
 }
+
+function toggleExpand(element) {
+    if (element.parentNode.classList.toggle('expanded')) {
+	element.style.transition = 'opacity 0.5s ease';
+	element.style.opacity = '0';
+    } else {
+	setTimeout(() => {
+	    element.style.transition = 'none';
+	    element.style.opacity = '1';
+	}, 500);
+    }
+}
+
+function addMaterialSymbol(element, symbol) {
+    const symbolSpan = document.createElement('span');
+    symbolSpan.className = 'material-symbols-outlined';
+    symbolSpan.textContent = symbol;
+    element.appendChild(symbolSpan);    
+}
```

Comment:
Need to add an id and remove it before appending, otherwise it stacks status elements for every manga page opened

Reply:
> otherwise it stacks status elements for every manga page opened

Have been clearing the `innerHTML` every time `showMangaDetails` is called.